### PR TITLE
Doc/shortcuts building and spawning

### PR DESCRIPTION
**Pull request by sschrijver**

### CODE REVIEW

File: docs/software_architecture/chapters/spawning-units.tex [hunk 1]
```diff
@@ -16,6 +16,8 @@ \subsection{Entity Panel} \label{EntityPanel}
 
 In order to capture click events, and trigger the unit spawning process, a MouseHandlerEntityPanel is used. When the player clicks on an entity, the mouse handler takes care of spawning the unit and subtracing the resource cost. The entity will only be ordered if the player has enough resources to pay for the unit, as shown below.
 
+Another way to spawn a unit is to press the shortcut key. This shortcut key will is shown in the top left badge of the unit panel.
```
Comment (MarkVlamPersie) [inline docs/software_architecture/chapters/spawning-units.tex]: This shortcut key **will** is shown

This word does not belong in the sentence.

File: docs/software_architecture/chapters/adding-buildings.tex [hunk 2]
```diff
@@ -3,12 +3,12 @@ \section{Adding buildings}
...
 
 \subsection{Building panel}
 The building panel, as displayed in \cref{fig:building-panel}, iterates over the buildings\_with\_textures enum, and for each entry it adds a UnitPanel to the Building Panel. UnitPanels are used for the EntityPanel (see \ref{EntityPanel}) as well as the Building Panel. These UnitPanels render an image, and contain objects of whatever it renders (buildings or entities). Unit panels are highlighted in \cref{fig:unit-panel}.
 
-A mouse handler of which a diagram is shown in \cref{fig:mousehandlerbuildingpanel} takes care of actually starting the process of positioning and placing a building.
+A mouse handler of which a diagram is shown in \cref{fig:mousehandlerbuildingpanel} or the key event handler takes care of actually starting the process of positioning and placing a building.
```
Comment (MarkVlamPersie) [inline docs/software_architecture/chapters/adding-buildings.tex]: +A mouse handler of which a diagram is shown in \cref{fig:mousehandlerbuildingpanel} or the key event handler takes care of actually

A mouse handler **,** of which a diagram is shown in \cref{fig:mousehandlerbuildingpanel}**,** or the key event handler takes care of actually 

added  commas to make the sentence more clear.

**Comment by MarkVlamPersie** [comment]
you can merge after those corrections IMO.

**Comment by sschrijver** [comment]
Made some changes, can you please check if its correct?

**Pull request merged**